### PR TITLE
add working favourite Button

### PR DESCRIPTION
**Pull request by kalipp1**
I added functionality to favorite button now we can change favorite state

### CODE REVIEW

File: src/components/common/ProductBox/ProductBox.js [hunk 1]
```diff
@@ -10,57 +10,81 @@ import {
...
-      {promo && <div className={styles.sale}>{promo}</div>}
-      <div className={styles.buttons}>
-        <Button variant='small'>Quick View</Button>
-        <Button variant='small'>
-          <FontAwesomeIcon icon={faShoppingBasket}></FontAwesomeIcon> ADD TO CART
-        </Button>
+const ProductBox = ({ id, name, price, promo, stars, isFavorite, isCompared }) => {
+  const dispatch = useDispatch();
+  const changeFavorite = (event, productId) => {
+    if (event.preventDefault) {
```
Comment (matwjablonski) [inline src/components/common/ProductBox/ProductBox.js:19]: why you need that check?

File: src/components/common/ProductBox/ProductBox.module.scss [hunk 2]
```diff
@@ -75,4 +75,19 @@
     justify-content: space-between;
     align-items: center;
   }
+  .favorite {
+    color: black;
```
Comment (matwjablonski) [inline src/components/common/ProductBox/ProductBox.module.scss:79]: colors only from variables, and don't use named colors

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: src/components/common/ProductBox/ProductBox.js (hunk 1 above)
Comment (kalipp1) [reply src/components/common/ProductBox/ProductBox.js]: All changes here done :D

**Review by NocnyZwierz — DISMISSED** [review]
Greate job!

**Review by NocnyZwierz — APPROVED** [review]
Good work!

**Review by NocnyZwierz — DISMISSED** [review]

**Review by NocnyZwierz — DISMISSED** [review]

**Review by NocnyZwierz — APPROVED** [review]

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**